sdk/appconfiguration/Azure.ApplicationModel.Configuration/src/Azure.Configuration.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>This is the Microsoft Azure Application Configuration Service client library</Description>
     <AssemblyTitle>Microsoft Azure.ApplicationModel.Configuration client library</AssemblyTitle>
-    <Version>1.0.0-preview.2</Version>
+    <VersionPrefix>1.0.0</VersionPrefix>
     <PackageTags>Microsoft Azure Application Configuration</PackageTags>
```

Comment:
Since we've released the preview for configuration, should this also have the `<PreReleaseVersionLabel/>` set to `preview.2` to match the released version?

Reply:
PreReleaseVersionLabel will be set at build queue time so if you want a preview.3 next then you would set that when you queue the build.